data/strings/strings.txt

```diff
@@ -22984,8 +22890,8 @@
 
   [pref_tts_speedcams_auto]
     comment = Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit
-    tags = android
-    en = Warn if there is a risk of exceeding the speed limit
+    tags = android,ios
+    en = Warn if exceeding the speed limit
```

Comment:
be = Папярэджваць калі хуткасць перавышана
  ru = Предупреждать при превышении скорости
  uk = Попереджати якщо швидкість перевищена

Reply:
nl = Waarschuw als de snelheidslimiet wordt overschreden

Reply:
> be = Папярэджваць калі хуткасць перавышана
> ru = Предупреждать при превышении скорости
> uk = Попереджати якщо швидкість перевищена

Cyrillic-based strings are shrinked...

![ios_speed_cam_prefs_ru1](https://github.com/organicmaps/organicmaps/assets/25888296/afe0e27a-5a32-44ec-84d9-9600d8417a81)

Reply:
> nl = Waarschuw als de snelheidslimiet wordt overschreden

@jimcarst: String is still a little bit too long, and it gets shrinked and truncated...

![ios_speed_cam_prefs_nl2](https://github.com/organicmaps/organicmaps/assets/25888296/96895576-eb61-43b1-9007-18f06e4ddf0d)

Can you please check if you can shorten it more?

Reply:
Not really. Extremely staccato solution would be:

nl = Waarschuw bij overschrijden snelheidslimiet

Reply:
> Not really. Extremely staccato solution would be:
> `nl = Waarschuw bij overschrijden snelheidslimiet`

@jimcarst: PTAL...

![ios_speed_cam_prefs_nl4](https://github.com/organicmaps/organicmaps/assets/25888296/c7ce8b59-2124-4bb9-ac59-080695e7e042)

Reply:
It looks like 47-50 characters is max possible limit to avoid ...
A comment to translators would be helpful.

Reply:
> It looks like 47-50 characters is max possible limit to avoid ...
> A comment to translators would be helpful.

Modified comment of string `comment = "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit`